frappe/www/printview.py

```diff
@@ -162,7 +162,13 @@ def get_rendered_template(
 		def get_template_from_string():
 			return jenv.from_string(get_print_format(doc.doctype, print_format))
 
-		if print_format.custom_format:
+		hook_func = frappe.get_hooks("get_print_format_template")
+		if len(hook_func):
```

Comment:
```suggestion
		if hook_func = frappe.get_hooks("get_print_format_template"):
```

Reply:
This should be `if hook_func := frappe.get_hooks("get_print_format_template"):` ?